src/app/SBE/detail/company/index.js

```diff
@@ -108,15 +108,15 @@ const Rep = ({ sbeCode, targetCode }) => {
         leftIcon={<FontAwesomeIcon icon={faDownload} />}
         colorScheme={'green'}
       >
-        {`Digital Jobs Participant Agreement`}
+        {`Digital Jobs Program Host Employer Subsidy Agreement`}
```

Comment:
I missed it in earlier pull requests but the name of the button cannot be hardcoded in the front end. It needs to be sent and set by the backend. It needs to be passed as a props (ie: name) and rendered dynamically.